var/spack/repos/builtin/packages/caliper/package.py

```diff
@@ -101,63 +103,94 @@ class Caliper(CMakePackage, CudaPackage, ROCmPackage):
     patch("for_aarch64.patch", when="target=aarch64:")
     patch("sampler-service-missing-libunwind-include-dir.patch", when="@2.9.0 +libunwind +sampler")
 
+    def _get_sys_type(self, spec):
+        sys_type = spec.architecture
+        if "SYS_TYPE" in env:
+            sys_type = env["SYS_TYPE"]
+        return sys_type
+
     def cmake_args(self):
+        options = []
+        return options
```

Comment:
You could just `return []` unless you're expecting to add arguments here.

Reply:
That, and the fact that cmake_args is defined twice in this package...

Reply:
Fixed